src/SoCreate.ServiceFabric.PubSub/SoCreate.ServiceFabric.PubSub.csproj

```
@@ -1,21 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Description>ServiceFabric.PubSubActors adds pub/sub behaviour to your Reliable Actors and Services in Service Fabric. How to use this package: https://github.com/loekd/ServiceFabric.PubSubActors/blob/master/README.md</Description>
+    <Description>SoCreate.ServiceFabric.PubSub adds pub/sub behaviour to your Reliable Actors and Services in Service Fabric. Documentation: https://service-fabric-pub-sub.socreate.it</Description>
     <Copyright>2019</Copyright>
-    <AssemblyTitle>ServiceFabric.PubSubActors</AssemblyTitle>
-    <Version>8.1.0</Version>
-    <Authors>Loek Duys</Authors>
-    <TargetFrameworks>net452;net46;netstandard2.0</TargetFrameworks>
+    <AssemblyTitle>SoCreate.ServiceFabric.PubSub</AssemblyTitle>
+    <Version>9.0.0</Version>
+    <Authors>Loek Duys, SoCreate</Authors>
+    <TargetFramework>netstandard2.0</TargetFramework>
     <PlatformTarget>x64</PlatformTarget>
     <Platform>x64</Platform>
-    <AssemblyName>ServiceFabric.PubSubActors</AssemblyName>
-    <PackageId>ServiceFabric.PubSubActors</PackageId>
-    <PackageTags>ServiceFabric;Service;Fabric;Actor;Pub;Sub;Reliable</PackageTags>
-    <PackageProjectUrl>https://github.com/loekd/ServiceFabric.PubSubActors</PackageProjectUrl>
-    <PackageLicenseUrl>https://github.com/loekd/ServiceFabric.PubSubActors/blob/master/LICENSE.md</PackageLicenseUrl>
+    <PackageId>SoCreate.ServiceFabric.PubSub</PackageId>
+    <PackageTags>ServiceFabric;Service;Fabric;PubSub;Pub;Sub;Reliable;Asynchronous;Messaging;Event</PackageTags>
+    <PackageProjectUrl>https://github.com/SoCreate/service-fabric-pub-sub</PackageProjectUrl>
+    <PackageLicenseUrl>https://github.com/SoCreate/service-fabric-pub-sub/blob/master/LICENSE</PackageLicenseUrl>
```

Change this to "https://raw.githubusercontent.com/SoCreate/service-fabric-pub-sub/master/LICENSE
```suggestion
    <PackageLicenseUrl>https://raw.githubusercontent.com/SoCreate/service-fabric-pub-sub/restructure/LICENSE</PackageLicenseUrl>
```